namespaces/decentraland.md

```diff
@@ -0,0 +1,58 @@
+---
+namespace: urn:decentraland
+github-org: https://github.com/decentraland
+contact-email: developers@decentraland.org
+---
+
+# Description
+
+Decentraland is a virtual reality platform powered by the Ethereum blockchain. 
+
+# Resolver structure
+
+The structure of the resolver is defined by a hierarchy of content address resolvers.
+
+```yml
+urn:decentraland:{protocol}:(...)
+```
+
+The hierarchy is defined by the `{protocol}` component of the URN.
+
+## The Ethereum resolvers
+
+Decentraland works in Ethereum mainnet, ropsten and Matic. There are 4 registered roots for blockchain assets:
```

Comment:
To map things on Matic dev we should use `Goerli` and `Mumbai`. I'd suggest rephrasing it as :

`Decentraland works in different blockchains and networks:`

Thinking at loud, Do we care if the asset is at `ropsten` or `mainnet`?  They should be indexed by different catalyst running for different environments

Reply:
> Thinking at loud, Do we care if the asset is at ropsten or mainnet? They should be indexed by different catalyst running for different environments

Yes, but the data should be unequivocally pointing to a specific network. Those need to be differentiated. Most people use the same wallet for ropsten and mainnet